.envrc.local.example

```diff
@@ -1 +1,3 @@
 export BLOCKFROST_KEY=xxxxxxxxxxx
+export ADDR_FROM=addr_test1xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+export ADDR_TO=addr_test1xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
Can we indicate somehow that these are just needed if you're running the demo?

And we also need to document that there needs to be a signing key stored in `infra/wallet/${ADDR_FROM}.skey` with funds on it

Reply:
I've updated the main branch so that keys can have any filename, to hopefully make it simpler to set up.

Reply:
Resolved